spec/models/card_spec.rb

```diff
@@ -31,8 +31,17 @@
     expect(card.errors.any?).to be true
   end
 
-  it "change cards number after success reviewing" do
-    succ_card = Card.pending.first
-    expect{succ_card.check_translation(succ_card.original_text)}.to change{Card.pending.count}.by(-1)
+  context "deals with database" do
+    let! (:card) { create(:card) }
+    before do
+      card.update_attribute(:review_date, Date.today - 1.day)
+    end
+
+    it "change cards number after success reviewing" do
+      succ_card = Card.pending.first
+      expect{succ_card.check_translation(succ_card.original_text)}.to change{Card.pending.count}.by(-1)
```

Comment:
Line is too long. [103/80]<br>Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.